### PR TITLE
Doc updates for mTLS feature and minor documentation bugs

### PR DESCRIPTION
**Pull request by shraddhabang**
### Issue

- [Document: Ingress specification sample is wrong and not matched to examples.](https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/3476)
- [Document Improvement: Cert-manager v1.5.4 is not compatible with supported EKS version](https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/3471)
- [Disable all SG creation and modification](https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/3468)
- [Is it possible to define different healthcheck protocol for a Service exposing two or more NodePorts ?
](https://github.com/kubernetes-sigs/aws-load-balancer-controller/issues/3346)

### Description

This PR includes the necessary documentation updates required for new mTLS support on Ingresses. It also has minor fixes for documentation issues mentioned above. 

### Testing
Already tested changes with ``make docs-preview``

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by oliviassss** [comment]
Please also verify the live doc rendering by `make docs-preview` if not already. Thanks

File: docs/guide/ingress/spec.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ spec:
       http:
         paths:
           - path: /*
+            pathType: Prefix
```
[the remark this answers is not in the document]
Comment (shraddhabang) [reply docs/guide/ingress/spec.md]: Updated Thanks.

**Comment by shraddhabang** [comment]
Yes. Already tested the change with ``make doc-preview``.  Added in PR description too. Thanks.

**Pull request merged**